nose/plugins/cover.py

```diff
@@ -121,6 +127,14 @@ def configure(self, options, conf):
             for pkgs in [tolist(x) for x in cover_packages]:
                 self.coverPackages.extend(pkgs)
         self.coverInclusive = options.cover_inclusive
+        self.coverInclusiveExcludes = []
+        if options.cover_exclude_file:
+            if isinstance(options.cover_exclude_file, (list, tuple)):
+                cover_exclude_file = options.cover_exclude_file
+            else:
+                cover_exclude_file = [options.cover_exclude_file]
+            for pkgs in [tolist(x) for x in cover_exclude_file]:
```

Comment:
Yep, that's to handle cases like '--cover-exclude-file foo,bar --cover-exclude-file bar'.

`extend()` (on the line below this), takes a list so this makes sense.